srcpkgs/python3-pgzero/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'python3-pgzero'
+pkgname=python3-pgzero
+version=1.2
+revision=1
+wrksrc="pgzero-${version}"
+build_style=python3-module
+pycompile_module="pgzero"
```

Comment:
also needs pgzrun.py

Reply:
How do I add this?

Reply:
When I build the package, it includes pgzrun in the generated package as `/usr/bin/pgzrun`... do I need to specify this explicitly?